dotcom-rendering/src/components/SelfHostedVideoPlayer.tsx

```diff
@@ -192,7 +192,7 @@ export const SelfHostedVideoPlayer = forwardRef(
 						videoStyles(width, height),
 						showSubtitles && subtitleStyles(subtitleSize),
 					]}
-					crossOrigin="anonymous"
+					crossOrigin={showSubtitles ? 'anonymous' : undefined}
```

Comment:
How does setting to `crossOrigin` to `anonymous` when there are subtitles work, but it fails if there aren't subtitles?

Reply:
Setting `crossOrigin` to `anonymous` is required for subtitles to work because the request for the video will only allow access to a sidecar vtt file if the video request uses CORS headers. CORS headers are not required if there are is subtitles file.

We experienced issues when we added `crossOrigin="anonymous"` because some users had the original video request cached before this change (not using CORS headers), then this cached request failed after this change. Therefore I think it's better if always make video requests with CORS header on production, and only locally omit CORS if it isn't required.